CHANGELOG.md

```diff
@@ -1,5 +1,48 @@
 # Database Tools Changelog
 
+## 100.5.3
+
+_Released 2022-06-XX_
+
+We are pleased to announce version 100.5.3 of the MongoDB Database Tools.
+
+This release contains a number of bug fixes and changes. Highlights include support for clustered collections in mongorestore, updating our Go version to address CVEs, and suported platform updates.
```

Comment:
Let's specifically mention we upgraded from 1.16.7 to 1.17.8.

s/suported/supported.

Reply:
Fixed both.